gym_xiangqi/piece.py

```diff
@@ -38,14 +38,22 @@ def __init__(self, color, row, col):
         self.state = ALIVE
         self.piece_width = PIECE_WIDTH
         self.piece_height = PIECE_HEIGHT
+        self.mini_piece_width = MINI_PIECE_WIDTH
+        self.mini_piece_height = MINI_PIECE_HEIGHT
         self.basic_image = None
         self.select_image = None
+        self.mini_image = None
+
+        # sound effect for piece movements
+        #sound = Sound()
```

Comment:
Are these suppose to be still here?

Reply:
These were just temporarily commented out for CI testing because doing so reduces the number of errors in CI. We will, however, need to do something with the sound card (or driver?) to solve the issue essentially.